roles/ansible/tower/manage-job-templates/README.md

```diff
@@ -60,7 +61,8 @@ ansible_tower:
     inventory: "Inventory1"
     project: "Project1"
     playbook: "playbooks/prep.yml"
-    credential: "Cred1"
+    credential:
```

Comment:
should this be `credentials` (with the trailing `s`) ?

Reply:
Good catch, thank you. I've updated.